packager/hls/base/master_playlist.cc

```diff
@@ -477,7 +484,7 @@ bool MasterPlaylist::WriteMasterPlaylist(
   std::string content = "#EXTM3U\n";
   AppendVersionString(&content);
```

Comment:
Can we write the tag after version string?

Reply:
changed